packages/search-ui-workplace-search-connector/src/__tests__/WorkplaceSearchAPIConnector.test.ts

```diff
@@ -331,6 +334,55 @@ describe("WorkplaceSearchAPIConnector", () => {
       `);
     });
 
+    it("will include meta header information correct for the browser context", async () => {
+      const state = {
+        ...DEFAULT_STATE,
+        searchTerm: "searchTerm",
+        current: 1,
+        resultsPerPage: 10,
+        sortDirection: "desc" as const,
+        sortField: "name",
+        filters: [
+          {
+            field: "title",
+            type: "all" as const,
+            values: ["Acadia", "Grand Canyon"]
+          },
+          {
+            field: "world_heritage_site",
+            values: ["true"],
+            type: "all" as const
+          }
+        ]
+      };
+
+      const queryConfig = {
+        current: 2,
+        resultsPerPage: 5,
+        sortDirection: "asc" as const,
+        sortField: "title",
+        filters: [
+          {
+            field: "date_made",
+            values: ["yesterday"],
+            type: "all" as const
+          }
+        ]
+      };
+
+      await subject(state, queryConfig);
+
+      expect(global.fetch).toHaveBeenCalledTimes(1);
+      const metaHeader = getLastFetchCall().headers["x-elastic-client-meta"];
+      expect(metaHeader).toEqual(
+        "ent=1.11.0-ws-connector,js=browser,t=1.11.0-ws-connector,ft=universal"
```

Comment:
I see that a similar test for onAutocomplete uses LIB_VERSION instead of hardcoded values. Maybe we could use it here as well to avoid updating tests in each version?

Reply:
thanks for spot! this should use `LIB_VERSION`